src/main/java/javax/jmdns/impl/DNSIncoming.java

```diff
@@ -289,7 +292,10 @@ private DNSQuestion readQuestion() {
         String domain = _messageInputStream.readName();
         DNSRecordType type = DNSRecordType.typeForIndex(_messageInputStream.readUnsignedShort());
         if (type == DNSRecordType.TYPE_IGNORE) {
-            logger.warn("Could not find record type: {}", this.print(true));
+            logger.warn("Could not find record type.");
```

Comment:
Please assign `_messageInputStream.readUnsignedShort()` to a variable (maybe `short wireType`) and log this value in the message to help identify the missing / unknown type.

Reply:
_typeForIndex_() method (before the if statement) which returned the _DNSRecordType_ has already logged (also on _WARN_ level) what you request here. So it would be redundant.

Reply:
You are right. Just saw it in your logs. (Actually I am not a fan of the logging within `DNSRecordType` but that's a code issue not an issue of this PR.)
This change would still result in two log entries:

```
WARN Could not find record type for index: XYZ
WARN Could not find record type.
```

I'd prefer something more like:
```
WARN Could not find record type. domain: 'abc' type: XYZ
```

This log way also indicates which device might have unknown RR types.